Server/src/database.ts

```diff
@@ -12,399 +12,507 @@ import { Subcategory } from './models/entities/Subcategory';
+    datapoint5.values = [1.026, 1.380, 1.972, 2.607, 2.779, 3.370, 3.748, 3.801, 3.948, 4.138, 4.290];
+    datapoint5.unitsId = unitsKMPS.id;
+    datapoint5.representationsId = reprUp.id;
+    await connection.manager.save(datapoint5);
+
+    let datapoint6 = new Datapoints();
+    datapoint6.id;
+    datapoint6.datasetId = dataset.id;
+    datapoint6.name = "Pressure";
+    datapoint6.values = [11.349, 17.617, 26.459, 36.599, 39.888, 57.567, 70.343, 71.672, 77.614, 85.091, 87.657];
+    datapoint6.unitsId = unitsGigapascal.id;
+    datapoint6.representationsId = reprP.id;
+    await connection.manager.save(datapoint6);
+
+    let datapoint7 = new Datapoints();
+    datapoint7.id;
+    datapoint7.datasetId = dataset.id;
+    datapoint7.name = "Specific Volume";
+    datapoint7.values = [0.3805, 0.3629, 0.3241, 0.2809, 0.2735, 0.2687, 0.2672, 0.2670, 0.2676, 0.2669, 0.2582];
+    datapoint7.unitsId = unitsCCG.id;
+    datapoint7.representationsId = reprV.id;
+    await connection.manager.save(datapoint7);
+
+    let datapoint8 = new Datapoints();
+    datapoint8.id;
+    datapoint8.datasetId = dataset.id;
+    datapoint8.name = "Density";
+    datapoint8.values = [2.628, 2.755, 3.086, 3.560, 3.657, 3.722, 3.743, 3.745, 3.737, 3.746, 3.873];
+    datapoint8.unitsId = unitsGCC.id;
+    datapoint8.representationsId = reprRho.id;
+    await connection.manager.save(datapoint8);
+
+    let datapoint9 = new Datapoints();
+    datapoint9.id;
+    datapoint9.datasetId = dataset.id;
+    datapoint9.name = "Compression Ratio";
+    datapoint9.values = [0.804, 0.770, 0.688, 0.602, 0.585, 0.577, 0.572, 0.570, 0.571, 0.570, 0.552];
+    datapoint9.unitsId = unitsNone.id;
+    datapoint9.representationsId = reprCompRatio.id;
+    await connection.manager.save(datapoint9);
+
+    let datapointcomments = new Datapointcomments();
+    datapointcomments.id;
+    datapointcomments.datasetId = dataset.id;
+    datapointcomments.comments = ["im1", "im1", "im1", "im1", "im1", "im1", "im1", "im1", "im1", "im1", "im1", "im1"];
+    await connection.manager.save(datapointcomments);
+
+    let datapoint10 = new Datapoints();
+    datapoint10.id;
+    datapoint10.datasetId = dataset2.id;
+    datapoint10.name = "Equivilence Ratio";
+    datapoint10.values = [0.0, 1.3, 1.6, 2.2, 2.5, 3.4];
+    datapoint10.unitsId = unitsNone.id;
+    datapoint10.representationsId;
+    await connection.manager.save(datapoint10);
+
+    let datapoint11 = new Datapoints();
+    datapoint11.id;
+    datapoint11.datasetId = dataset2.id;
+    datapoint11.name = "Cell Width";
+    datapoint11.values = [25.5, 30.0, 36.5, 60.0, 50.5, 102.0];
+    datapoint11.unitsId = unitsMM.id;
+    datapoint11.representationsId;
+    await connection.manager.save(datapoint11);
+
+    console.log("All data entered");
+
+    // End element insertion
+
+    /**
+     * Testing finding an element
+     */
+    // let savedAnime = await connection.manager.find(AnimeRank, { title: "Promised Neverland"});
```

Comment:
Are these commented out snippets of code for testing necessary? Do you think we'll need them in production eventually (master)?

Or have you kept them simply for convenience, that you may use them in the future when updating the DB?

Reply:
Yes these are very much needed. While the terminology here is for testing, specifically they will form the basis of multiple queries to be created in #53. As #53 progresses, these will be removed